### PR TITLE
Onevnet multinet provider

### PR DESCRIPTION
**Pull request by tuxmea**
onevnet and addressrange types and providers.
tested on opennebula virtualbox image

### CODE REVIEW

File: lib/puppet/type/onevnet_addressrange.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  end
+
+  newproperty(:protocol) do
+    desc "Type of the addressrange. Valid values: IP4, IP6, IP4_6"
+    newvalues(:ip4, :ip6, :ip4_6)
+  end
+
+  newproperty(:ip_start) do
+    desc "Base ip address for IPv4 networks."
+    #validate do |value|
```
Comment (rwaffen) [inline lib/puppet/type/onevnet_addressrange.rb]: this doesn't do anything?

**Comment by rwaffen** [comment]
looks okay to me... but onevnet_addressrange has some commented code which seems to do nothing...

File: lib/puppet/type/onevnet_addressrange.rb (hunk 1 above)
Comment (tuxmea) [inline lib/puppet/type/onevnet_addressrange.rb]: Wrong! It defines properties!

**Comment by tuxmea** [comment]
I assume you are referring to the onevnet_addressrange type.
At the moment this lacks validation. I will work on this next.

**Pull request merged**